packages/embark/src/cmd/cmd_controller.js

```diff
@@ -461,22 +461,53 @@ class EmbarkController {
       function (callback) {
         engine.init({}, callback);
       },
-      function (callback) {
-        engine.startService("libraryManager").installAll((err) => callback(err ? err : null));
-      },
```

Comment:
Same here.. the installation of deps through the library manager has been introduced for a reason. It's unclear why this is removed to "fix" the graph command.

Reply:
all this cmd was outdated, and was updated to replicate the run. so stuff like this start service for libraryManager was outdated